### ISSUE TITLE
[help] Can't use it locally: `Error! Failed to complete request to /api/hello: Error: socket hang up`

### ISSUE THREAD

**Issue by sixclones** [vercel-community/deno#98]
Hi,

I'm trying to use deno as a runtime but it seems that I'm missing something in my setup.

## My project

```
project
|__web
    api
    |__hello.ts
    .vercel.json
```

`hello.ts`
```ts
export default () => new Response(`Hello, from Deno v${Deno.version.deno}!`);
```

`vercel.json`
```json
{
	"version": 2,
	"functions": {
		"api/**/*.[jt]s": {"runtime": "vercel-deno@1.1.0"}
	}
}
```

## Vercel

I created a project in Vercel, linked it to my Git repository and defined `web` as my root directory. Next, inside my project I ran `vercel link`.

## On my side

Deno `v1.18.1` is installed, vercel CLI `v23.1.2` is installed and NodeJS `v16.13.0` is installed.

<br />

When I run `vercel dev`, everything's good.

```bash
> Ready! Available at http://localhost:3000
```

But then when I go `http://localhost:3000/api/hello`, I got:

```
502: BAD_GATEWAY
Code: NO_RESPONSE_FROM_FUNCTION
ID: dev1::dev1::tk09g-1643464806192-eb798a5b7597
```

and from the terminal

```bash
Error! Failed to complete request to /api/hello: Error: socket hang up
```

<br />

I tried on two computers (one using WSL2, one using macOS), result is the same.

The thing is that when I deploy on Vercel, the API is working: https://numbered-tech.vercel.app/api/hello.

<br />

What I'm missing?

Thanks

**Comment by rafaelcanical** [comment]
Same thing happening to me

**Comment by wit03** [comment]
the problem may cause by using Router from Oak. I tried to deploy both with Router implemented and without and the error only occurs on the Router implemented version.

**Comment by JiDai** [comment]
I have exactly the same problem with a similar simple example 

```
export default async () => new Response('pong');
```

Did you find a solution or workaround to make it work ? 
I don't know how to debug or understand the error...

**Comment by kidonng** [comment]
You can replace `dev-server.ts` found in the following directories with the patched one in #102:

- macOS: `~/Library/Caches/com.vercel.cli/dev/builders/node_modules/vercel-deno`
- Windows: `%APPDATA%\Roaming\npm\node_modules\vercel\node_modules\vercel-deno`

Note these are not exact location (and I may be wrong).
